students/11th/daseulsong/main.html

```diff
@@ -0,0 +1,292 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8" />
+    <meta name="viewport" content="width=device-width" />
+    <title>main-page</title>
+    <link
+      href="https://fonts.googleapis.com/css2?family=Open+Sans:wght@400;600;700&display=swap"
+      rel="stylesheet"
+    />
+    <link href="style/common.css" rel="stylesheet" type="text/css" />
+    <link href="style/main.css" rel="stylesheet" type="text/css" />
+  </head>
+  <script
+    src="https://kit.fontawesome.com/f2f9d53b4c.js"
+    crossorigin="anonymous"
+  ></script>
+  <script src="js/main.js" defer></script>
+  <body>
+    <header>
+      <div class="header__container">
```

Comment:
```suggestion
      <div class="headerContainer">
```
클래스명 컨벤션은 camel case나 snake case로 통일시켜주세요

Reply:
snake case로 모두 통일하였습니다.